log-manager/src/main/java/io/airlift/log/LoggingConfiguration.java

```diff
@@ -155,4 +156,16 @@ public LoggingConfiguration setLegacyLoggerImplementationEnabled(boolean enabled
         this.legacyLoggerImplementationEnabled = enabled;
         return this;
     }
+
+    public String getLogAnnotationFile()
+    {
+        return logAnnotationFile;
+    }
+
+    @Config("log.annotation-file")
+    public LoggingConfiguration setLogAnnotationFile(String logAnnotationFile)
```

Comment:
We could annotate with `@FileExists`. The config system will validate it exists if not null.